### DISCUSSION TITLE
how to customize/localized the error messages returned by resteasy

### DISCUSSION THREAD

**Discussion by git4x** [resteasy/resteasy#3023] [Q&A]
For example, if the URL path is not correct, resteasy returns the error message "Could not find resource for full path: blah blah blah...", our project wants to do some customization, but get no luck to make it.
Please help, thanks!

**Comment by jamezp** [comment]
This is a bit complicated I suppose as it would require a custom build of RESTEasy. RESTEasy using the JBoss Logging and the [JBoss Logging Tools](https://jboss-logging.github.io/jboss-logging-tools/) for internationalization. However, it currently does not have any translations done as we do not have the resources to create or maintain them.
